frontend-react/src/hooks/network/History/__mocks__/DeliveryHooks.ts

```diff
@@ -1,3 +1,16 @@
 import * as DeliveryHooks from "../DeliveryHooks";
```

Comment:
As far as I know, we're continuing to follow the nomenclature patterns of Jest's auto mocking capabilities: https://jestjs.io/docs/manual-mocks#mocking-user-modules

Reply:
Took a moment, but I found it. This document was what we wrote up during a norming session, and because we're unclear on whether we want to auto-mock or not in the long run, and we've followed the pattern for a bit (albeit wrongly in a few places like using `__mocks__` for `msw` handlers) I think the outcome was that we'd keep using the mocks folders with matching file names as if we were to transition to auto-mocking